src/components/Tag/Tag.module.css

```diff
@@ -0,0 +1,99 @@
+@import '../../design-tokens/mixins.css';
+
+/*------------------------------------*\
+    #TAGS
+\*------------------------------------*/
+
+/**
+ * The rounded rectangle container that houses information.
+ * 1) Grabs colors from whichever variant is selected.
+ * 2) Rounds the corners of the tag.
+ */
+.tag {
+  width: max-content;
+  max-width: 20rem;
+  padding-top: var(--eds-size-half);
+  padding-bottom: var(--eds-size-half);
+  padding-left: var(--eds-size-1-and-half);
+  padding-right: var(--eds-size-1-and-half);
+
+  display: inline-flex;
+  align-items: center;
+
+  border: var(--tag-secondary-color) solid var(--eds-theme-border-width); /* 1 */
+  border-radius: 9999px; /* 2 */
+
+  /** 
+   * TODO: (dcwither) when adding SvgIcon to EDS, include this in tailwind font 
+   * size macro.
+   */
```

Comment:
I think we can remove this comment since we're not using tailwind anymore.

Reply:
What about the line under it `--svg-icon-size-default: 1.25rem;` ? Are we using that variable in icons?

Reply:
As per https://github.com/chanzuckerberg/edu-design-system/pull/941#discussion_r842993194, will remove comment and change the css variable name